go.mod

```diff
@@ -82,6 +82,7 @@ require (
 	github.com/morikuni/aec v1.0.0 // indirect
 	github.com/opencontainers/go-digest v1.0.0 // indirect
 	github.com/opencontainers/image-spec v1.1.0-rc6 // indirect
+	github.com/openfga/language v0.0.0-20240205122832-cf938f8254bd // indirect
```

Comment:
Is this needed? It is already referenced above as:
```
github.com/openfga/language/pkg/go v0.0.0-20240131004817-1e763f816993
```

Let's update the version there instead

`go get -u github.com/openfga/language/pkg/go`